crossing-training.lic

```diff
@@ -89,6 +96,7 @@ class CrossingTraining
     if trash_nouns.any? { |noun| /\b#{noun}/i =~ GameObj.left_hand.noun } && !@equipment_manager.is_listed_item?(left_hand)
       dispose_trash(left_hand)
     end
+    dump_junk if DRRoom.room_objs.size > 9 && @settings.dump_junk
```

Comment:
I'd prefer to see these two `if`s moved into guard clauses in `dump_junk`. What do you think?